homeassistant/components/unifi/controller.py

```diff
@@ -155,6 +193,36 @@ def event_update(self):
                 self.unifi_config = unifi_config
                 break
 
+        options = self.config_entry.options
+
+        if CONF_BLOCK_CLIENT in self.unifi_config:
+            options[CONF_BLOCK_CLIENT] = self.unifi_config[CONF_BLOCK_CLIENT]
+
+        if CONF_DONT_TRACK_CLIENTS in self.unifi_config:
+            options[CONF_DONT_TRACK_CLIENTS] = self.unifi_config[
```

Comment:
please don't write directly to the options object of the config entry. It might end up getting saved if another config entry is saved.